py/lib/lowering_neon.py

```diff
@@ -0,0 +1,83 @@
+# Transformations apply to all SSE, AVX and AVX512.
```

Comment:
Please just remove these comments related to SSE (or refer to Neon, if you feel the comments are needed)